.gitignore

```diff
@@ -30,3 +31,14 @@ venv
 .venv
 env
 .env
+
+# Generated files
+debian/.debhelper
+debian/lutris
+obj-x86_64-linux-gnu/*
+debian/debhelper-build-stamp
+debian/files
+debian/lutris.debhelper.log
+debian/lutris.postinst.debhelper
+debian/lutris.prerm.debhelper
+debian/lutris.substvars
```

Comment:
Don't do this. Simply call `debclean`, and you have a clean folder again.

Reply:
Ah, I didn't see that in the contributing documentation and I didn't want to mistakenly commit those files.  Is there some reason not to have this in the .gitignore?  Seems like a good way to prevent unintentional commits that then have to be cleaned up.